* Install "doc/burgerspace-server.6" only when installing the server
* Use "$datarootdir" for FDO specified paths
  This is recommended by Debian, as $datadir could be specified to lie
  on a different file system, whereas icons/desktop/pixmaps are specified
  by freedesktop.org and are based on $datarootdir
  -> https://blogs.gnome.org/hughsie/2014/06/16/datarootdir-v-s-datadir/
* Do not install LICENSE
  The fact that the source is licensed under the GPL doesn't require installing
  the COPYING file along.
* Nested sound files in "$pkgdatadir", i.e., $(datadir)/$(PACKAGE)

--- a/Makefile.am
+++ b/Makefile.am
@@ -2,12 +2,13 @@
 
 SUBDIRS = src
 
-man_MANS = doc/burgerspace.6 doc/burgerspace-server.6
+man_MANS = doc/burgerspace.6
+if BUILD_NETWORK
+man_MANS += doc/burgerspace-server.6
+endif
 
-docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
 doc_DATA = \
 	AUTHORS \
-	COPYING \
 	NEWS \
 	README \
 	THANKS
@@ -17,6 +18,7 @@
 	$(RPMBUILD) -ta $(distdir).tar.gz
 
 EXTRA_DIST = \
+	COPYING \
 	bootstrap \
 	autogen.sh \
 	$(PACKAGE).spec \
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -74,10 +74,10 @@
 	PKGSOUNDDIR=$(srcdir)/sounds ./$(PACKAGE)
 
 
-desktopentrydir = $(datadir)/applications
+desktopentrydir = $(datarootdir)/applications
 desktopentry_DATA = $(PACKAGE).desktop
 
-pixmapdir = $(datadir)/pixmaps
+pixmapdir = $(datarootdir)/pixmaps
 pixmap_DATA = images/$(PACKAGE).png
 
 
@@ -193,7 +193,7 @@
 	images/digit8.xpm \
 	images/digit9.xpm
 
-pkgsounddir = $(datadir)/sounds/$(PACKAGE)
+pkgsounddir = $(pkgdatadir)/sounds
 pkgsound_DATA = \
 	sounds/ingredient-bounces.wav \
 	sounds/ingredient-in-plate.wav \
